libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/GlobalLogLevel.scala

```diff
@@ -1,20 +1,20 @@
 // Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.platform.sandbox
+package com.daml.logging
```

Comment:
I’d probably move this In the CLI opts library instead. You really want to use this together with the CLI parser whereas the contextualized logger can be used perfectly fine without either of these and the other way around, you don’t need the contextualized logger for this to be useful.

Reply:
Valid point, will do that